manifest.json

```diff
@@ -1,17 +1,30 @@
 {
     "spark_version": "2.4.0",
```

Comment:
shouldn't that be updated to `2.4.3` too?

Reply:
Couldn't find where this property is used or visible. Updated anyway though.